<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>okno3</class>
 <widget class="QDialog" name="okno3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 64, 64);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(123, 26, 255, 255), stop:1 rgba(187, 28, 226, 255));</string>
   </property>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>10</y>
      <width>190</width>
      <height>190</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 95px;
background-color: rgb(64, 64, 64);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(123, 26, 255, 255), stop:1 rgba(187, 28, 226, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>-15</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 45px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00272727, y2:1, stop:0.164773 rgba(134, 26, 250, 255), stop:0.647727 rgba(154, 27, 241, 255), stop:1 rgba(121, 239, 255, 246));</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>44</y>
     <width>61</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background-color: rgb(64, 64, 64);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_10" native="true">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>43</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(157, 27, 239);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>21</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 45px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.00272727, y2:1, stop:0.6875 rgba(157, 27, 239, 255), stop:1 rgba(121, 239, 255, 246));</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_11" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(64, 64, 64);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_12" native="true">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>55</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(64, 64, 64);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_13" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>68</y>
     <width>71</width>
     <height>84</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_14" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>413</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 45px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(150, 27, 238, 255), stop:1 rgba(167, 27, 235, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_15" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(157, 27, 239);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_16" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>21</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(157, 27, 239);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_17" native="true">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>373</y>
     <width>40</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(64, 64, 64);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_19" native="true">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>375</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(157, 27, 239);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_18" native="true">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>405</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background-color: rgb(64, 64, 64);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_20" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>407</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 64, 64);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_21" native="true">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>434</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(148, 27, 244);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_22" native="true">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>455</y>
     <width>61</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(127, 26, 253, 255), stop:1 rgba(143, 27, 246, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_23" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>380</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_24" native="true">
   <property name="geometry">
    <rect>
     <x>-31</x>
     <y>420</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background-color: rgb(127, 26, 253);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_25" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background-color: rgb(127, 26, 253);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_26" native="true">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>414</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(64, 64, 64);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_27" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>79</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0,0);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-10</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,0);
font: 600 28pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Уже уходите?</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_32" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0,0);</string>
   </property>
   <widget class="QWidget" name="widget_28" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: qlineargradient(spread:pad, x1:0.458, y1:0, x2:0.48, y2:1, stop:0 rgba(157, 27, 239, 255), stop:1 rgba(134, 163, 249, 255));</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_29" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(157, 27, 239);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_30" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>29</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: qlineargradient(spread:pad, x1:0.506, y1:0, x2:0.469, y2:1, stop:0 rgba(150, 68, 242, 255), stop:0.715084 rgba(134, 163, 249, 250), stop:1 rgba(121, 239, 255, 246));

</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_33" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>120</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,0);</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0,0);
color: rgb(253, 253, 253);</string>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <zorder>widget_29</zorder>
   <zorder>widget_28</zorder>
   <zorder>widget_30</zorder>
   <zorder>widget_33</zorder>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>321</y>
     <width>190</width>
     <height>190</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 95px;
background-color: rgb(64, 64, 64);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_35" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0,0);</string>
   </property>
   <widget class="QWidget" name="widget_41" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>34</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0.466, y1:0, x2:0.469, y2:1, stop:0 rgba(141, 121, 246, 255), stop:1 rgba(121, 239, 255, 246));</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_34" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: qlineargradient(spread:pad, x1:0.466, y1:0, x2:0.469, y2:1, stop:0 rgba(157, 27, 239, 255), stop:1 rgba(141, 121, 246, 246));</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_37" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>40</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: qlineargradient(spread:pad, x1:0.458, y1:0, x2:0.48, y2:1, stop:0 rgba(157, 27, 239, 255), stop:1 rgba(121, 239, 255, 255));</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_38" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0.466, y1:0, x2:0.469, y2:1, stop:0 rgba(141, 121, 246, 255), stop:1 rgba(121, 239, 255, 246));</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_36" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: qlineargradient(spread:pad, x1:0.458, y1:0, x2:0.48, y2:1, stop:0 rgba(157, 27, 239, 255), stop:1 rgba(134, 163, 249, 255));</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_39" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: qlineargradient(spread:pad, x1:0.458, y1:0, x2:0.48, y2:1, stop:0 rgba(157, 27, 239, 255), stop:1 rgba(134, 163, 249, 255));</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_42" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>14</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0,0);
color: rgb(253, 253, 253);</string>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <zorder>widget_37</zorder>
   <zorder>widget_41</zorder>
   <zorder>widget_38</zorder>
   <zorder>widget_36</zorder>
   <zorder>widget_39</zorder>
   <zorder>widget_34</zorder>
   <zorder>widget_42</zorder>
  </widget>
  <widget class="QWidget" name="widget_31" native="true">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>400</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0.466, y1:0, x2:0.469, y2:1, stop:0 rgba(141, 121, 246, 255), stop:1 rgba(121, 239, 255, 246));
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(0, 0, 0, 0);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>69</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0,0);
color: rgb(253, 253, 253);</string>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>69</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0,0);
color: rgb(253, 253, 253);</string>
   </property>
   <property name="text">
    <string>Нет</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_40" native="true">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>405</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(157, 27, 239);</string>
   </property>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>widget_6</zorder>
  <zorder>widget</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_5</zorder>
  <zorder>widget_9</zorder>
  <zorder>widget_7</zorder>
  <zorder>widget_11</zorder>
  <zorder>widget_12</zorder>
  <zorder>widget_10</zorder>
  <zorder>widget_13</zorder>
  <zorder>widget_8</zorder>
  <zorder>widget_14</zorder>
  <zorder>widget_15</zorder>
  <zorder>widget_16</zorder>
  <zorder>widget_18</zorder>
  <zorder>widget_20</zorder>
  <zorder>widget_23</zorder>
  <zorder>widget_19</zorder>
  <zorder>widget_21</zorder>
  <zorder>widget_24</zorder>
  <zorder>widget_25</zorder>
  <zorder>widget_26</zorder>
  <zorder>widget_22</zorder>
  <zorder>widget_27</zorder>
  <zorder>widget_32</zorder>
  <zorder>widget_35</zorder>
  <zorder>widget_31</zorder>
  <zorder>label_3</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>widget_40</zorder>
  <zorder>widget_17</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
